### PR TITLE
Restore archlinux support

### PR DESCRIPTION
**Pull request by bastelfreak**
this got removed from the params.pp. Was that an accident or on purpose?

### CODE REVIEW

File: spec/classes/repo_spec.rb [hunk 1]
```diff
@@ -7,32 +7,26 @@
...
         it { is_expected.to contain_class(contained_class) }
         it { is_expected.to_not contain_class(not_contained_class) }
 
         it { is_expected.to contain_class('influxdb::repo') }
-        it { is_expected.to compile }
+        it { is_expected.to compile.with_all_deps }
 
         # ordering tests
         it { is_expected.to contain_class(contained_class).that_comes_before('Class[influxdb::repo]') }
-
```
Comment (n1tr0g) [inline spec/classes/repo_spec.rb]: why did you remove this test ?

Comment (bastelfreak) [inline spec/classes/repo_spec.rb]: sorry, this got lost during refactoring

**Comment by bastelfreak** [comment]
should be ready for merge now. I also applied the rubocop config from Vox Pupuli to clean up the spec tests.
